Library/Formula/bitrise.rb

```diff
@@ -0,0 +1,26 @@
+require "language/go"
+
+class Bitrise < Formula
+  desc "Command-line automation tool"
+  homepage "https://github.com/bitrise-io/bitrise"
+  url "https://github.com/bitrise-io/bitrise/archive/0.9.7.tar.gz"
+  sha256 "cecd0cd112941c6b6f9d6cd334ea6a31aef1123af89e17c3950fbc8e8dbe5de2"
+
+  depends_on "go" => :build
+
+  def install
+    mkdir_p buildpath/"src/github.com/bitrise-io"
+    ln_s buildpath, buildpath/"src/github.com/bitrise-io/bitrise"
+
+    ENV["GOPATH"] = "#{buildpath}/Godeps/_workspace:#{buildpath}"
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", "_bin/bitrise"
```

Comment:
Ohh, I see, sure, if you think that's a better way to do it it's certainly possible.

P.S.: I copied the structure from https://github.com/Homebrew/homebrew/blob/master/Library/Formula/docker-swarm.rb